lib/imap.php

```diff
@@ -37,7 +37,7 @@ class OC_User_IMAP extends \OCA\user_external\Base {
 	 * @param boolean $stripeDomain (whether to stripe the domain part from the username or not)
 	 * @param boolean $groupDomain (whether to add the usere to a group corresponding to the domain of the address)
 	 */
-	public function __construct($mailbox, $port = null, $sslmode = null, $domain = null, $stripeDomain = true, $groupDomain = false) {
+	public function __construct($mailbox, $port = null, $sslmode = null, $domain = null, $stripeDomain = false, $groupDomain = false) {
```

Comment:
the thing is the current behaviour is to stripe the domain...
that's why I've put this to true by default, so we don't break anything for current setups...

Reply:
also we only apply it if a restrictig domain is specified...

Reply:
I wouldn't set it as default either, but as we do it this way in the existing versions, we shouldn't change it now, because it would break existing setups... so I've reverted your last commit, hope it's clear now why :)